src/pages/learn/index.tsx

```diff
@@ -0,0 +1,943 @@
+              and how they relate to Ethereum? And what is Ethereum anyway?{" "}
+            </p>
+            <p>
+              <Link to="/what-is-ethereum/">
+                Check out our introduction to Ethereum
+              </Link>
+              .
+            </p>
+            <p>
+Not only can Ethereum do what Bitcoin does (transfer money globally), it’s capable of much more – anyone can deploy code onto the network. And because it’s so flexible, any computer program can run on Ethereum.
+            </p>
+            <CardGrid>
+              <Card
+                title="What is Ethereum?"
+                description="If you are new, start here. Learn the basics of crypto why Ethereum matters."
+              >
+                <>
+                  <CardImage>
+                    <GatsbyImage
+                      image={getImage(data.whatIsEth)}
+                      alt="Illustration of a person peering into a bazaar, meant to represent Ethereum.1"
+                    />
+                  </CardImage>
+                  <ButtonLink to="/what-is-ethereum/">
+                    What is Ethereum?
+                  </ButtonLink>
+                </>
+              </Card>
+              <Card
+                title="What is ETH?"
+                description="The currency of the Ethereum network is called ether or ETH."
+              >
+                <>
+                  <CardImage>
+                    <GatsbyImage
+                      image={getImage(data.eth)}
+                      alt=""
+                    />
+                  </CardImage>
+                  <ButtonLink to="/eth/">What is ETH?</ButtonLink>
+                </>
+              </Card>
+              <Card
+                title="Where can I get ETH?"
+                description="There are many ways to aquire ETH depending on your location."
+              >
+                <>
+                  <CardImage>
+                    <GatsbyImage
+                      image={getImage(data.impact)}
+                      alt=""
+                    />
+                  </CardImage>
+                  <ButtonLink to="/get-eth/">Where can I get ETH?</ButtonLink>
+                </>
+              </Card>
+            </CardGrid>
+
+            <AdditionalReadingHeader>
+              Additional reading on Ethereum
+            </AdditionalReadingHeader>
+            <DocsContainer>
+              <DocLink to="/smart-contracts/">
+                What are smart contracts?
+              </DocLink>
+              <DocLink to="/developers/docs//intro-to-ethereum/">
+                A technical introduction to Ethereum
+              </DocLink>
+              <DocLink to="/web3/">What is Web3?</DocLink>
+              <DocLink
+                to="https://www.youtube.com/watch?v=WSN5BaCzsbo"
+                isExternal
+              >
+                Decentralizing Everything
```

Comment:
We can bend the rules here and sentence case this for visual consistency.

```suggestion
                Decentralizing everything
```